templates/fragments/_bind.erb

```diff
@@ -1,6 +1,6 @@
 <% require 'ipaddr' -%>
 <%- if @bind -%>
-<%- @bind.sort.map do |address_port, bind_params| -%>
+<%- @bind.sort_by { |address_port, bind_params| address_port.split('.').map{ |octet| octet.to_i} }.map do |address_port, bind_params| -%>
```

Comment:
Perhaps this sort code should first partition the keys into two arrays, one containing only the things that look like IP addresses and one containing everything else, then sort both, concatenate them again and process as before.

And what about IPv6 addresses?